freqtrade/freqtradebot.py

```diff
@@ -909,7 +909,7 @@ def handle_timedout_limit_buy(self, trade: Trade, order: Dict) -> bool:
 
         logger.info('Buy order %s for %s.', reason, trade)
 
-        if safe_value_fallback(corder, order, 'remaining', 'remaining') == order['amount']:
+        if safe_value_fallback(corder, order, 'filled', 'filled') == 0.0:
```

Comment:
```suggestion
        # 'remaining' is not filled properly by all exchanges (ftx)
        # (and the definition of 'remaining' is unclear)
        # so 'filled' is used instead here and below
        filled_value = safe_value_fallback(corder, order, 'filled', 'filled') == 0.0:
        if filled_value == 0.0:
```
-- not sure about the correctness of the comment
-- change the name of the variable, maybe you'll find a better one (also used below)
-- comparison of 2 floats for equality again, do not like it...

Reply:
I did verify this for binance, kraken and FTX - and all return filled directly (in the info record - but for kraken as `vol_exec`).

In the end, the comparison before was the same - comparing `remaining == amount` can have the same rounding problem you think we got - but it's a good point, we can use "isclose" here ... :)